src/types/general.ts

```diff
@@ -41,12 +41,18 @@ type ProxyMapping = {
   port?: string
 }
 
+type ApiTokens = {
```

Comment:
2 issues here:
1. Should be singular. 
2. `AccessToken` is a better name than API token.